nodes.py

```diff
@@ -1343,6 +1342,7 @@ def INPUT_TYPES(s):
                     "negative": ("CONDITIONING", ),
                     "latent_image": ("LATENT", ),
                     "denoise": ("FLOAT", {"default": 1.0, "min": 0.0, "max": 1.0, "step": 0.01}),
+                    "batch_behavior": (["randomize", "fixed", "increment", "decrement"],),
```

Comment:
this would need to be under `optional` to avoid breaking presaved api workflows (and use `="some_default"` on the func params)